dowhy/causal_refuters/add_unobserved_common_cause.py

```diff
@@ -4,6 +4,9 @@
 import pandas as pd
 import scipy.stats
 
+import tqdm
+from tqdm.notebook import tqdm
```

Comment:
we need tqdm.notebook only if user is running the code in a notebook. Otherwise we need tqdm.

better to use tqdm.auto. see this example: https://github.com/tqdm/tqdm/pull/1218